src/main/java/org/minions/devfund/chelo/Squarelotron.java

```diff
@@ -0,0 +1,194 @@
+     */
+    public void rotateRight(int numberOfTurns) {
+
+        final int majicTurns = 4;
+        Squarelotron arrayUpsideDown = new Squarelotron(this.size);
+        Squarelotron squarelotronRotate = new Squarelotron(this.size);
+        Squarelotron squarelotronAux = initializeArray(this.size);
+        int realTurns;
+
+        realTurns = numberOfTurns % majicTurns + majicTurns;
+
+        for (int turns = 1; turns <= realTurns % majicTurns; turns++) {
+            for (int ring = 1; ring <= (this.size / 2) + (this.size % 2); ring++) {
+                arrayUpsideDown = getUpsideDownFlipArray(arrayUpsideDown, squarelotronAux.squarelotron, ring);
+            }
+
+            for (int i = 0; i < this.size; i++) {
+                for (int j = 0; j < this.size; j++) {
+                    squarelotronRotate.squarelotron[j][i] = arrayUpsideDown.squarelotron[i][j];
+                }
+            }
+            squarelotronAux = squarelotronRotate;
+        }
+        this.squarelotron = squarelotronRotate.squarelotron;
+    }
+
+    /**
+     * A method to upside down flip.
+     * @param arrayResult the resultant array
+     * @param array the given array
+     * @param ring the ring
+     * @return new array flipped
+     */
+    Squarelotron getUpsideDownFlipArray(final Squarelotron arrayResult, final int[][] array, int ring) {
+        int indexX;
+        int indexY;
+
+        indexX = ring - 1;
+        for (int m = ring - 1; m < this.size - ring + 1; m++) {
+            arrayResult.squarelotron[this.size - ring][indexX] = array[ring - 1][m];
+            indexX++;
+        }
+
+        indexX = ring - 1;
+        indexY = ring - 1;
+        for (int n = ring - 1; n < this.size - ring + 1; n++) {
+            arrayResult.squarelotron[indexX][indexY] = array[this.size - ring][n];
+            indexY++;
+        }
+
+        indexX = this.size - ring - 1;
+        indexY = ring - 1;
+        for (int j = ring; j < this.size - ring; j++) {
+            arrayResult.squarelotron[indexX][indexY] = array[j][ring - 1];
+            indexX--;
+        }
+
+        indexX = this.size - ring - 1;
+        indexY = this.size - ring;
+        for (int k = ring; k < this.size - ring; k++) {
+            arrayResult.squarelotron[indexX][indexY] = array[k][this.size - ring];
+            indexX--;
+        }
+        return arrayResult;
+    }
+
+    /**
+     * A method to print the squarelotron content.
+     * @param array squarelotron
+     */
+    void getArray(final Squarelotron array) {
+        for (int i = 0; i < array.squarelotron.length; i++) {
+            for (int j = 0; j < array.squarelotron.length; j++) {
+                System.out.print("[" + array.squarelotron[i][j] + "]");
+            }
+            System.out.println(" ");
+        }
+    }
+
+    /**
+     * Method to return squarelotron array.
+     * @return array
+     */
+    public int[][] getSquarelotron() {
+        int[][] aux = squarelotron;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "aux". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)